cmd/create.go

```diff
@@ -410,8 +410,8 @@ func createHost(provider, name, region, zone, projectID, userData, inletsPort st
 	} else if provider == "civo" {
 		return &provision.BasicHost{
 			Name:       name,
-			OS:         "811a8dfb-8202-49ad-b1ef-1e6320b20497",
-			Plan:       "g2.small",
+			OS:         "12745392-15c7-4140-925d-441fe7ae57fd",
```

Comment:
Can you extract the OS string into a const, just above the return statement? Don't put it at the top of the file.

`const Ubuntu2004 = "12745392-15c7-4140-925d-441fe7ae57fd"`

Reply:
Done.

const CivoUbuntu2004ID= ...